src/css/styles.css

```diff
@@ -13,7 +13,7 @@ body, html {
 }
 
 nav {
-  margin-bottom: 12%;
+  /* margin-bottom: 12%; */
 }
 
```

Comment:
We should go back and get rid of the commented out code before merging this branch